(#23480) Fixed problem with wrong directory path

Mintormo opened the pull request.

akien-mga commented on editor/filesystem_dock.cpp:
```diff
@@ -1602,6 +1602,8 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> p_selected)
+				if (!fpath.ends_with("/"))
+					fpath = fpath + "/";
 			}
 			make_script_dialog_text->config("Node", fpath + "new_script.gd", false);
```
The proper fix would be to use `fpath.plus_file("new_script.gd")` instead of `fpath + "new_script.gd"`.

Mintormo replied:
Good. I will try to fix it. Thanks for idea.

akien-mga commented:
That should work, but see my comment for a better fix. It would be nice if you could also amend your commit to use a more explicit commit log - the commit log should tell us in words what the commit is doing. When reading the `git log` locally, "Fixed #23480" is not enough information to know what this commit does, as we're forced to go on GitHub and paste the issue number to find out.

See https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#format-your-commit-logs-with-readability-in-mind and http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html for amending the commit.

Commit:
(#23480) Wrong directory path in filesystem dock when creating script.

When creating script and file selected in filesystem dock,
between file name and directory path there is no "/" symbol.
This commit fix that bug.

akien-mga commented:
Thanks!
